config/database.yml

```diff
@@ -0,0 +1,25 @@
+# SQLite. Versions 3.8.0 and up are supported.
```

Comment:
Please follow comments by trainers for the previous tutorial
- gitignore this file
- create new file is named `database.yml.example` in `config/` folder
- config db use MySQL (refs: https://edgeguides.rubyonrails.org/configuring.html#configuring-a-mysql-or-mariadb-database)